src/contconic.jl

```diff
@@ -34,11 +36,11 @@ function _lin1test(solver::Function, config::TestConfig, vecofvars::Bool)
     end
 
     c = MOI.addconstraint!(instance, MOI.VectorAffineFunction([1,1,1,2,2], [v;v[2];v[3]], ones(5), [-3.0,-2.0]), MOI.Zeros(2))
-    @test MOI.get(instance, MOI.NumberOfConstraints{vecofvars ? MOI.VectorOfVariables : MOI.VectorAffineFunction{Float64},MOI.Nonnegatives}()) == 1
+    @test MOI.get(instance, MOI.NumberOfConstraints{_vovf(vecofvars),MOI.Nonnegatives}()) == 1
```

Comment:
Can't we just go
```julia
if vecofvars
    @test MOI.get(instance, MOI.NumberOfConstraints{MOI.VectorOfVariables,MOI.Nonnegatives}()) == 1
else
    @test MOI.get(instance, MOI.NumberOfConstraints{MOI.VectorAffineFunction{Float64},MOI.Nonnegatives}()) == 1
end
```
and ditto with everywhere else this happens.

Reply:
For this one, I am not sure it is worth it. The readability of that part is less important and since the part with `addconstraint!` is readable, he will already know what we want to do here. Having 5 lines for such trivial branching may clutter the code a bit.

Reply:
I agree that it's not necessary to be verbose here. I'd prefer the inline `?` version over the helper function, however.

Reply:
That's a good compromise.